cmake/modules/ConfigCMake.cmake

```diff
@@ -17,7 +17,15 @@
 #
 include ("${CMAKE_SOURCE_DIR}/cmake/ConfigDefault.cmake")
 
-# A "ConfigUser.cmake" in the source tree overrides the defaults.
+# A "GMTSettings.cmake" in the source tree overrides the basic defaults.
+if (EXISTS ${GMT_SOURCE_DIR}/cmake/GMTSettings.cmake)
+	include (${GMT_SOURCE_DIR}/cmake/GMTSettings.cmake)
+else ()	# Default is to copy the support data into the build shared dir
+	set (COPY_GSHHG TRUE)
+	set (COPY_DCW TRUE)
```

Comment:
Move these two lines to `cmake/ConfigDefault.cmake`? It seems most default values are set in that file.